schema/schema2025-03-19.yaml

```
@@ -1,41 +1,197 @@
-  /api/insult/{id}:
+  /api/categories/:
     get:
-      operationId: insult_retrieve
+      operationId: list_categories
+      description: |-
+        List all available insult categories. <br><br> Returns a dictionary containing a help text and a dictionary of available categories
+                where the key is the category key and the value is the category name. <br><br>This endpoint can be used as a reference for other endpoints where the category is a required field or desired filter. <br><br><<br>strong>Note:</strong> All Endpoints will accept either the category key or name as a valid input for filtering or creating insults.
       parameters:
-      - in: path
-        name: id
+      - name: page
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)